src/theory/arith/theory_arith.h

```diff
@@ -152,6 +156,11 @@ class TheoryArith : public Theory {
   ArithPreprocess d_arithPreproc;
   /** The theory rewriter for this theory. */
   ArithRewriter d_rewriter;
+
+  /**
+   * Proofs enabled?
+   */
+  bool proofsEnabled() const;
```

Comment:
Maybe we should generalize this so that it is in `theory.h`.

Reply:
Ah. Good point! Added to the `Theory` class in b9332110c.